test/integration/environment_inherritance.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -e
```

Comment:
With or without this flag, the tests will correctly verify. The main difference is that without this flag, the tests will run properly and a failure in them will be caught by the check for their return code. If we were to have the flag, the whole script would fail and prove would not be able to properly tell which test failed, it would simply say that the entire script failed.